src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -89,12 +89,6 @@ export class CalciteRadioButton implements LabelableComponent, CheckableFormComp
   @Watch("name")
   nameChanged(): void {
     this.checkLastRadioButton();
-    const currentValue: HTMLInputElement = this.rootNode.querySelector(
-      `input[name="${this.name}"]:checked`
-    );
-    if (!currentValue?.value) {
-      this.uncheckAllRadioButtonsInGroup();
```

Comment:
It seems to work fine for me. Can you confirm @eriklharper?

Reply:
Sure thing, I'll try that locally real quick.  If there's not a test yet for that I'll add one.

Reply:
Just out of curiosity, why is this being removed?

Reply:
It was failing tests when left in with the changes herre.

Reply:
This should be fine to remove.  I'm blanking on the original purpose of this code, but it seems like we don't need to do this.  Just because the name changes doesn't mean we have to change the checked state of the group.

Reply:
LGTM! 👍

Reply:
Cool, yeah I don't see a problem removing this.